resources/meals.json

```diff
@@ -1054,6 +1054,103 @@
       "tomatoes",
       "parmesanCheese"
   ]
+},
+
+"italian cannoli crepes": {
+  "Display Name": "Italian Cannoli Crepes",
+  "Search Terms": [
+    "italian",
+    "crepe",
+    "cannoli",
+    "sweet"
+  ],
+  "Breakfast": 0,
+  "Lunch": 0,
+  "Dinner": 0,
+  "Dessert": 1,
+  "Ingredients": [
+    "flour",
+    "sugar",
+    "salt",
+    "heavyCream",
+    "eggs",
+    "milk",
+    "ricottaCheese",
+    "lemon",
+    "butter",
+    "pistachio"
+  ]
+},
+
+"BomboloniDoughnut": {
```

Comment:
This should be bomboloniDoughnut to stick with camelcase

Reply:
Right. I will fix it right away.